v63002/u_inref/gtm8165.csh

```diff
@@ -0,0 +1,27 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Testing YottaDB can support fractional timeout values to the millisecond for
+# LOCK, OPEN and $gtm_tpnotacidtime
+# Not testing READ since it is demonstrated in r120/readtimeout subtest, or JOB which
+# is logistically too complicated to test
+#
+$gtm_tst/com/dbcreate.csh mumps 1 >>& dbcreate.out
+setenv gtm_tpnotacidtime .123
```

Comment:
Need this echoed in reference file as say "# Set gtm_tpnotacidtime to .123 seconds"